### PR TITLE
Tashas cauldron of fixes

### PR DESCRIPTION
**Pull request by Aeristoka**
I think this is a pretty fair number at least (not sure I caught all of them) of the new Tasha's stuff.

### CODE REVIEW

File: src/common/settings.js [hunk 1]
```diff
@@ -459,9 +459,9 @@ const character_settings = {
         "type": "bool",
         "default": false
     },
-    "fighter-giant-might": {
-        "title": "Fighter: Giant Might",
-        "description": "Activate Giant Might to get advantage on Strength checks and saving throws and deal 1d6 extra damage",
+    "fighter-giants-might": {
```
Comment (kakaroto) [inline src/common/settings.js:462]: no reason to change the name of the option, that would invalidate whatever setting the user had before

File: src/dndbeyond/content-scripts/character.js [hunk 2]
```diff
@@ -432,9 +453,16 @@ function rollItem(force_display = false, force_to_hit_only = false, force_damage
...
+        if (character.hasClassFeature("Blessed Strikes") &&
+            character.getSetting("cleric-blessed-strikes", false)) {
+            damages.push("1d8");
+            damage_types.push("Blessed Strikes");
+        }
         // Bard's Psychic blades;
         if (character.hasClassFeature("Psychic Blades") &&
-            character.getSetting("bard-psychic-blades", false)) {
+            character.getSetting("bard-psychic-blades", false) &&
+            character.hasClass("Bard")) {
```
Comment (kakaroto) [inline src/dndbeyond/content-scripts/character.js:465]: Can you have a class feature of Psychic Blades without being a Bard?  Is that some new feature in Tasha's ?

File: src/dndbeyond/content-scripts/character.js [hunk 3]
```diff
@@ -726,6 +774,13 @@ function rollAction(paneClass, force_to_hit_only = false, force_damages_only = f
                 damages.push("1d8");
                 damage_types.push("Radiant");
             }
+
+            // Cleric Blessed Strikes
+            if (character.hasClassFeature("Blessed Strikes") &&
+            character.getSetting("cleric-blessed-strikes", false)) {
```
Comment (kakaroto) [inline src/dndbeyond/content-scripts/character.js:780]: some indentation issues here

Comment (kakaroto) [reply src/dndbeyond/content-scripts/character.js:780]: Blessed Strikes applies to "weapon attacks and cantrips", so it should apply on rollItem and rollSpell. I'm unsure about rollAction. Wouldn't actions be non-weapon attacks ?

**Review by kakaroto — CHANGES_REQUESTED** [review]
Looking good, just small tweaks.

File: src/dndbeyond/content-scripts/character.js (hunk 3 above)
Comment (Aeristoka) [reply src/dndbeyond/content-scripts/character.js:780]: This is based on a conversation we had some time back about us deciding weapon attacks included natural weapons (which would then include fists/claws/tails/etc.)

File: src/dndbeyond/content-scripts/character.js (hunk 2 above)
Comment (Aeristoka) [reply src/dndbeyond/content-scripts/character.js:465]: Yeah, thank WotC for this crap:
Rogue Psychic Blades: https://www.dndbeyond.com/sources/tcoe/rogue#PsychicBlades
Bard College of Whispers Psychic Blades: https://www.dndbeyond.com/classes/bard#CollegeofWhispers

Just, ya know, named the same dang stupid exact same name...

File: src/common/settings.js (hunk 1 above)
Comment (Aeristoka) [reply src/common/settings.js]: Good point, fixed

File: src/dndbeyond/content-scripts/character.js (hunk 2 above)
Comment (kakaroto) [reply src/dndbeyond/content-scripts/character.js:465]: oh cool, so.. who cares about multiclassers!

Comment (Aeristoka) [reply src/dndbeyond/content-scripts/character.js:465]: They'll still be good, as it catches only for Bard here, and they'd have that if they multi-classed bards & rogues.

Unless you're mouthing off at WotC, if so I AGREE!

Comment (kakaroto) [reply src/dndbeyond/content-scripts/character.js:465]: yeah, I was mouthing off 😁 
But if someone has both rogue psychic blades and bard psychic blades then they're SOL.

Comment (Aeristoka) [reply src/dndbeyond/content-scripts/character.js:465]: Shouldn't be, the part for the Rogue is elsewhere in the code

Comment (kakaroto) [reply src/dndbeyond/content-scripts/character.js:465]: eh, right, I'm stupid, I was thinking of rolling "Psychic Blades" not of rolling something while also having the feat.
How about you're a Bard, but not yet 3rd level, or not of the right college, and you have rogue psychic blades.
(note: I'm just thinking of issues that I have no intentions of fixing. There's an option for that feat anyway, just annoyed at WotC crap)

Comment (Aeristoka) [reply src/dndbeyond/content-scripts/character.js:465]: yeah, that could be an issue... they'd have to have it toggled on...

I say screw it as well, the DM and Player should know what the things be

**Pull request merged**